prboom2/src/MUSIC/portmidiplayer.c

```diff
@@ -207,6 +232,9 @@ static void pm_shutdown (void)
     Pt_Sleep (DRIVER_LATENCY * 2);
     #endif
 
+	// stop all sound, in case of hanging notes
+    pm_killnotes(0);
```

Comment:
Couldn't you just call `pm_stop()` here?

Reply:
Good point, I'll do that.

Reply:
It also seems insufficient - hanging notes still occur. I'll investigate further.